docker-compose-deployment.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    env_file:
+      - db.env
+    ports:
+      - "127.0.0.1:5432:5432"
+    volumes:
+      - /home/ec2-user/db:/var/lib/postgresql/data
```

Comment:
should `/home/ec2-user/db` be configurable? seems tailored to EC2/

Reply:
@dvrkni  let's change the name to ec2 deployment to be very open sourcy :)

Reply:
that works for me as well :-)